abci/types/util.go

```diff
@@ -23,7 +23,7 @@ func (v ValidatorUpdates) Len() int {
 	return len(v)
 }
 
-// XXX: doesn't distinguish same validator with different power.
+// Less doesn't distinguish the same validator with different power.
```

Comment:
I wouldn't remove the tag here. This should be a FIXME, TODO?

Reply:
i don't know what reason is, let me rollback the comment